src/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -45,80 +45,86 @@ public function __construct( ImageAttachmentSchema $image_attachment_schema ) {
 	public function get_properties() {
 		return [
-			'key'                 => [
+			'key'                         => [
 				'description' => __( 'Unique identifier for the item within the cart.', 'woo-gutenberg-products-block' ),
 				'type'        => 'string',
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
-			'id'                  => [
+			'id'                          => [
 				'description' => __( 'The cart item product or variation ID.', 'woo-gutenberg-products-block' ),
 				'type'        => 'integer',
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
-			'quantity'            => [
+			'quantity'                    => [
 				'description' => __( 'Quantity of this item in the cart.', 'woo-gutenberg-products-block' ),
 				'type'        => 'integer',
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
-			'quantity_limit'      => [
+			'quantity_limit'              => [
 				'description' => __( 'The maximum quantity than can be added to the cart at once.', 'woo-gutenberg-products-block' ),
 				'type'        => 'integer',
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
-			'name'                => [
+			'name'                        => [
 				'description' => __( 'Product name.', 'woo-gutenberg-products-block' ),
 				'type'        => 'string',
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
-			'short_description'   => [
+			'short_description'           => [
 				'description' => __( 'Product short description in HTML format.', 'woo-gutenberg-products-block' ),
 				'type'        => 'string',
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
-			'description'         => [
+			'description'                 => [
 				'description' => __( 'Product full description in HTML format.', 'woo-gutenberg-products-block' ),
 				'type'        => 'string',
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
-			'sku'                 => [
+			'sku'                         => [
 				'description' => __( 'Stock keeping unit, if applicable.', 'woo-gutenberg-products-block' ),
 				'type'        => 'string',
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
-			'low_stock_remaining' => [
+			'low_stock_remaining'         => [
 				'description' => __( 'Quantity left in stock if stock is low, or null if not applicable.', 'woo-gutenberg-products-block' ),
 				'type'        => [ 'integer', 'null' ],
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
-			'backorders_allowed'  => [
+			'backorders_allowed'          => [
 				'description' => __( 'True if backorders are allowed past stock availability.', 'woo-gutenberg-products-block' ),
 				'type'        => [ 'boolean' ],
 				'context'     => [ 'view', 'edit' ],
 				'readonly'    => true,
 			],
-			'sold_individually'   => [
+			'show_backorder_notification' => [
```

Comment:
If this can be renamed safely (badge vs notification) then I'd vote for that. Though I can see that `notification` is used for this in core api / backend.

Reply:
Fixed in 4c26498c0f063763e0592ccb698cf08a2280eea1 too.